server/src/main.rs

```diff
@@ -28,17 +28,22 @@ impl Pricing for ArrowPricing {
     /// Get pricing for a given query.
     async fn get_pricing(
         &self,
-        request: tonic::Request<pricing_grpc::PricingRequest>,
+        request: tonic::Request<pricing_grpc::PricingRequests>,
     ) -> Result<tonic::Response<pricing_grpc::PricingResponse>, tonic::Status> {
-        grpc_info!("Received request: {:?}", request);
         let query = request.into_inner();
-        grpc_debug!("Getting pricing for query: {:?}", query);
-        let price = get_pricing(query);
-        grpc_debug!("Pricing computed: {}", price);
-        let response = pricing_grpc::PricingResponse { price };
-        grpc_debug!("Returning response: {:?}", response);
-        grpc_info!("Successfully computed pricing; returning response");
-        Ok(tonic::Response::new(response))
+        info!("Received pricing request");
+        let prices = get_pricing(query);
+        match prices {
+            Ok(prices) => {
+                let total_price = prices.iter().fold(0.0, |acc, x| acc + x);
+                let response = pricing_grpc::PricingResponse { price: total_price };
```

Comment:
Might we want to display per-leg cost breakdown to the customer? Is that too much?

Reply:
[18a8e13](https://github.com/Arrow-air/svc-pricing/pull/17/commits/18a8e139860ee66001dda4ad87e896e02fff3eac)